### PR TITLE
feat: adding missing flux functions

### PR DESCRIPTION
**Pull request by russorat**
Closes #

Adds the missing flux functions from https://docs.influxdata.com/influxdb/v2.0/reference/flux/stdlib/all-functions/

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 52 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1602)

**Review by TCL735 — APPROVED** [review]

File: src/shared/constants/fluxFunctions.ts [hunk 1]
```diff
@@ -267,7 +408,98 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
       'array.from(rows: [{_time: 2020-01-01T00:00:00Z, _value: "foo"},{_time: 2020-01-02T00:00:00Z, _value: "bar"}])',
     category: 'Inputs',
     link:
-      'https://docs.influxdata.com/influxdb/v2.0/reference/flux/stdlib/experimental/array/from/',
+      'https://docs.influxdata.com/influxdb/cloud/reference/flux/stdlib/experimental/array/from/',
```
Comment (sanderson) [inline src/shared/constants/fluxFunctions.ts:411]: The array package was promoted from experimental. You can also change the package on line 405 to `array` (won't let me make suggestions on a line that wasn't edited)

```suggestion
      'https://docs.influxdata.com/influxdb/cloud/reference/flux/stdlib/array/from/',
```

File: src/shared/constants/fluxFunctions.ts [hunk 2]
```diff
@@ -962,530 +1386,866 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
...
   },
   {
-    name: 'experimental.addDuration',
+    name: 'elapsed',
     args: [
       {
-        name: 'd',
-        desc: 'The duration to add.',
+        name: 'unit',
+        desc: 'The unit time to return. Defaults to `1s`.',
```
Comment (sanderson) [inline src/shared/constants/fluxFunctions.ts:1396]:
```suggestion
        desc: 'The unit of time to return. Defaults to `1s`.',
```

**Review by sanderson — APPROVED** [review]
Found just some minor typos and submitted suggestions. Thanks for this @russorat.

**Pull request merged**